source/Octopus.Tentacle/Tentacle.exe.nlog

```diff
@@ -34,7 +34,7 @@
     <logger name="LogFileOnlyLogger" writeTo="octopus-log-file" final="true" />
     <logger name="*" minlevel="Info" maxLevel="Warn" writeTo="stdout" />
     <logger name="*" minlevel="Error" writeTo="stderr" />
-    <logger name="*" minlevel="Info" writeTo="octopus-log-file" />
+    <logger name="*" minlevel="Trace" writeTo="octopus-log-file" />
```

Comment:
Do we want to keep this at Trace level?

Reply:
Ahh, no. I'm having issues with the service not starting correctly sometimes hence this hack!!